### ISSUE TITLE
Why is Iron marked as deprecated?

### ISSUE THREAD

**Issue by xpe** [flosse/rust-web-framework-comparison#57]
I don't think it is deprecated, since:
* I see recent commits at https://github.com/iron/iron/commits/master
* I did not find any official mention of deprecation

(Generally speaking, to mark something as deprecated, it is useful to point to a blog post or some official source for confirmation.)

**Comment by knpwrs** [comment]
I was about to open this issue as well. I'm new to the rust world but it seems like iron is still under active development.

**Comment by spejamchr** [comment]
Looks like https://github.com/iron/iron/pull/576 marked iron as "not actively maintained", and then https://github.com/iron/iron/pull/580 reverted that.

**Comment by flosse** [comment]
...so feel free to create a PR :)
BTW: Thanks for contributing!

**Comment by xpe** [comment]
@flosse Would it be fair to say you can make the change in about as much time as it takes us to make the PR?

**Comment by xpe** [comment]
I submitted the PR request for two main reasons: (1) Correcting mistakes feels good; (2) I didn't want anyone to be dissuaded from trying Iron based on incorrect information.

## Meta Commentary

Now, I'd like to make a some meta-comments about the "...so feel free to create a PR :)" comment, above. 

I would encourage people to question the following statement, which seems to be adopted as dogma by many people in open source:

> every change to open source code should be in the form of a pull request (even if the change is trivial)

I have found that pragmatism and weighing many perspectives is useful. Here are some other viewpoints to consider besides "the submit a PR if you care" cliche:

* Strict adherence to rules that are not categorically true can be problematic. (For example, the dogma I mention above.) Adherence to inflexible, poorly fitting rules can often lead to responses that are not suited to particular situations and people.

* If making a simple change yourself is faster than asking someone else to do it for you, why not do it yourself?

* The author of content is ultimately responsible for its accuracy. So, if something you write is not true (because of a mistake, or the passage of time), and you become aware of it, is it not your responsibility to fix once you become aware of it?

* There is a big difference between one person asking for a feature to be developed in an open source project, such as NumPy, and a person offering a simple, factual correction to what is, essentially, a blog post.

* Under what circumstances does it make to ask other people to correct your mistakes?

I know that some of this is rather pointed. I'm not trying to be prickly. I appreciate the work @flosse put into this write-up. I simply want to call awareness to some broader issues. Personally, I've probably just seen the phrase "feel free to create a PR" one too many times. When I see it written, it often seems to correspond with a lack of awareness around the broader, more important issues around interacting as a community.

To summarize, you could say I'm advocating for flexibility over dogma.
